.github/workflows/causal_lm_cpp.yml

```diff
@@ -810,6 +810,17 @@ jobs:
           && ./build/samples/cpp/visual_language_chat/visual_language_chat ./llava_v1_6_mistral_7b_ov/ monalisa.jpg
           <<< $'Who drew this painting?\nWhen did the painter live?'
         timeout-minutes: 4
+      - name: Download and convert InternVL2 model and an image
+        run: |
+          source ./ov/setupvars.sh
+          optimum-cli export openvino --model OpenGVLab/InternVL2-4B ./internvl2_4b_ov/ --trust-remote-code
+          wget https://llava-vl.github.io/static/images/monalisa.jpg
```

Comment:
this picture is downloaded 2 times above.
Actually, we can keep only first download as next ones save file as monalisa.jpg.1, monalisa.jpg.2 etc

Reply:
Fixed